### PR TITLE
[ONNX] Add HF CNN fp32 model tests

### PR DESCRIPTION
**Pull request by jinchen62**
| items        |    tests |   model-run |   onnx-import |   torch-mlir |   iree-compile |   inference |
|:-------------|---------:|------------:|--------------:|-------------:|---------------:|------------:|
| total-count  | 530 |     500     |       500     |            129 |        73     |     16     |

full list (530): https://gist.github.com/jinchen62/cdf54ef8ed725fcce9d6fa18ecbfa058
failed on model-run (30): https://gist.github.com/jinchen62/1b23c9c227b772c7e7641fad39aa6ea9
failed on torch-mlir/iree-compile (411): https://gist.github.com/jinchen62/0ad6931179badcba0860f2da01b3cc21
failed on inference (73): https://gist.github.com/jinchen62/c870373258c5a1c71c975f81258463de

commands:
`python ./run.py  --tolerance 0.001 0.001 -c path/to/torch-mlir/build -i path/to/iree-build --cachedir path/to/cache -f onnx -g models --mode onnx --report -j 12 --testsfile model_list.txt`

### CODE REVIEW

**Review by AmosLewis — APPROVED** [review]
LGTM

File: e2eshark/onnx/models/darknet53/model.py
Comment (ScottTodd) [inline e2eshark/onnx/models/darknet53/model.py:1]: This conflicts with https://github.com/nod-ai/SHARK-TestSuite/blob/main/e2eshark/onnx/models/DarkNet53/model.py on case insensitive file systems like Windows.

As long as both files exist using different casings, I can't get my repo into a clean state:

```bash
D:\dev\projects\SHARK-TestSuite\iree_tests (main -> upstream)
λ git reset --hard HEAD
HEAD is now at a2508f30 Fix Path instruction in README.md (#321)

D:\dev\projects\SHARK-TestSuite\iree_tests (main -> upstream)
λ git pull
Already up to date.

D:\dev\projects\SHARK-TestSuite\iree_tests (main -> upstream)
λ git status
On branch main
Your branch is up to date with 'upstream/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   ../e2eshark/onnx/models/darknet53/model.py

no changes added to commit (use "git add" and/or "git commit -a")

D:\dev\projects\SHARK-TestSuite\iree_tests (main -> upstream)
λ git diff
diff --git a/e2eshark/onnx/models/darknet53/model.py b/e2eshark/onnx/models/darknet53/model.py
index f221670c..54fc5be6 100644
--- a/e2eshark/onnx/models/darknet53/model.py
+++ b/e2eshark/onnx/models/darknet53/model.py
@@ -22,7 +22,7 @@ session = onnxruntime.InferenceSession("model.onnx", None)
 # Even if model is quantized, the inputs and outputs are
 # not, so apply float32
 # Get and process the image
-img_ycbcr = setup_test_image(288, 288)
+img_ycbcr = setup_test_image()

 model_input_X = to_numpy(img_ycbcr)

```

Comment (ScottTodd) [reply e2eshark/onnx/models/darknet53/model.py:1]: (the repo has a `darknet53` folder and a `DarkNet53` folder, but those are identical paths on Windows)

Comment (jinchen62) [reply e2eshark/onnx/models/darknet53/model.py:1]: Do you think we should rename the new one or remove the new one?

Comment (ScottTodd) [reply e2eshark/onnx/models/darknet53/model.py:1]: If the only difference is
```diff
-img_ycbcr = setup_test_image(288, 288)
+img_ycbcr = setup_test_image()
```
then I would just delete one of them

Comment (jinchen62) [reply e2eshark/onnx/models/darknet53/model.py:1]: I verified that the difference between these two models is the input shape. Pls review https://github.com/nod-ai/SHARK-TestSuite/pull/322.

**Pull request merged**